Formula/protoc-gen-gogofaster.rb

```diff
@@ -0,0 +1,30 @@
+class ProtocGenGogofaster < Formula
+  desc "Protocol Buffers for Go with Gadgets"
+  homepage "https://github.com/gogo/protobuf"
+  url "https://github.com/gogo/protobuf/archive/v1.3.1.tar.gz"
+  sha256 "5184f06decd681fcc82f6583976111faf87189c0c2f8063b34ac2ea9ed997236"
+  license "BSD-3-Clause"
+  head "https://github.com/gogo/protobuf.git"
+
+  depends_on "go" => :build
+  depends_on "protobuf"
+
+  def install
+    system "go", "build", *std_go_args, "-ldflags", "-s -w", "./protoc-gen-gogofaster"
+  end
+
+  test do
+    protofile = testpath/"proto3.proto"
+    protofile.write <<~EOS
+      syntax = "proto3";
+      package proto3;
+      message Request {
+        string name = 1;
+        repeated int64 key = 2;
+      }
+    EOS
+    system "protoc", "--gogo_out=.", "proto3.proto"
```

Comment:
```suggestion
    system "#{bin}/protoc", "--gogo_out=.", "proto3.proto"
```

Reply:
@SMillerDev What does `#{bin}` reference? The issue with this line is that `--gogo_out` should have been `--gogofaster_out`, which was a copy pasta error on my part. I didn't add `#{bin}` because I didn't do that with https://github.com/Homebrew/homebrew-core/pull/60022

Reply:
It references the bin directory of the formula install. It makes sure we're not accidentally calling a system program.

Reply:
Makes sense, that is what caused me to think it was working locally afterall

Reply:
Oh actually, this doesn't work, because `protoc` is actually a dependency (`protobuf`), this formula is enacted with `--gogofaster_out=.` section of the command.